src/server/dfly_bench.cc

```diff
@@ -767,10 +779,11 @@ int main(int argc, char* argv[]) {
 
   uint32_t thread_key_step = 0;
   const uint32_t qps = GetFlag(FLAGS_qps);
-  const int64_t interval = qps ? 1000000000LL / qps : 0;
+  const int64_t interval = qps ? 1000'000'000LL / qps : 0;
```

Comment:
```suggestion
  const int64_t interval = qps ? 1'000'000'000LL / qps : 0;
```
Since you already touched it :rofl: